### PR TITLE
WEB-2801: Missing fullname at the beginning of email course creation

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** 
When customers are not users yet, e.i. when coach does not already exist, the full name field remains empty and the greeting makes no sense.

Solution: 
if name of user is exist > `Dear <fullname>, ` else `Hi there, ` 

**JIRA:** https://liv-it.atlassian.net/browse/WEB-2801

**Merge deadline:** 31 may 2018

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] tag reviewer
- [ ] tag reviewer

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

**Review by avkoval — APPROVED** [review]

File: themes/courses.labster.com/lms/templates/emails/enroll_email_ccx_coach_enrolledmessage.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ from django.conf import settings
 from django.utils.translation import ugettext as _
 %>
 
-${_("Dear {full_name}").format(full_name=full_name)}
+${_("Hi there,")}
```
Comment (TamoshaytisV) [inline themes/courses.labster.com/lms/templates/emails/enroll_email_ccx_coach_enrolledmessage.txt]: Will there be space after the comma in resulting sentence?

**Review by TamoshaytisV — APPROVED** [review]

File: themes/courses.labster.com/lms/templates/emails/enroll_email_ccx_coach_enrolledmessage.txt (hunk 1 above)
Comment (lukyfebrianto) [reply themes/courses.labster.com/lms/templates/emails/enroll_email_ccx_coach_enrolledmessage.txt]: @TamoshaytisV  no

**Comment by lukyfebrianto** [comment]
After talked with Emma we decide to use name of user if exist. 
Sorry, please review it again
^ @TamoshaytisV , @avkoval

**Review by avkoval — APPROVED** [review]
I don't think we need additional spaces after } but everything else looks well

**Review by TamoshaytisV — APPROVED** [review]

**Pull request merged**